ChangeLog.md

```diff
@@ -17,6 +17,8 @@ See docs/process.md for how version tagging works.
 
 Current Trunk
 -------------
+ - Add support for override config variables using environment variables.  Any
+   config variables `FOO` can be overridden by `EM_FOO` in the environment.
```

Comment:
`variables` => `variable`

Reply:
Also, maybe `.emscripten config file` instead of `config` to clarify what is meant?